fe/fe-core/src/main/java/com/starrocks/catalog/TabletStatMgr.java

```diff
@@ -215,6 +221,13 @@ private void updateLakeTabletStat() {
         }
     }
 
+    private void adjustStatUpdateRows(long tableId, long totalRowCount) {
+        BasicStatsMeta meta = GlobalStateMgr.getCurrentAnalyzeMgr().getBasicStatsMetaMap().get(tableId);
+        if (meta != null) {
+            meta.setUpdateRows(totalRowCount);
+        }
+    }
```

Comment:
Oh guy, can't modify updateRows directly, it's a delta, not real row count. auto statistics collect job depend on the value